lib/errors/validation-error.ts

```diff
@@ -34,8 +34,18 @@ export enum ValidationErrorItemOrigin {
 }
 
 /**
- * Validation Error Item
  * Instances of this class are included in the `ValidationError.errors` property.
+ * 
+ * | parameter | description |
```

Comment:
You wrote this as markdown but it should be jsdoc

Reply:
I've tried jsdoc but looks like it's accept markdown instead of jsdoc, please kindly check :) @ephys

Reply:
`@param` is officially supported by esdoc which we're using so it should work. If it doesn't it means there is something else going on. Either way breaking we should definitely not break the jsdoc.